lib/src/sync/timeouts.dart

```diff
@@ -40,6 +41,11 @@ class Timeouts {
         _handler.timeouts.parseSetPageLoadTimeoutResponse);
   }
 
+  /// Gets the current timeout values.
```

Comment:
Should be a getter?
(Unlss we specifically want it to match the underlying W3C API.)

And the operations above should probably have been setters, but they didn't have getters, which was an argument against that. Should they have getters now?
Does it make sense to get the individual timeouts?)
Would it make sense to set all the timeouts in one batch?

The assymmetry is irksome :)

Reply:
I didn't want to change the existing API. We could introduce a new setter for all the timeouts though. The W3C API would support setting them all in one go:
https://www.w3.org/TR/webdriver2/#set-timeouts
https://github.com/google/webdriver.dart/blob/master/lib/src/handler/w3c/timeouts.dart#L21

I can do that if you think it's worth it.